src/components/tutorial-meta/components/variant-list/index.tsx

```diff
@@ -0,0 +1,58 @@
+// @TODO uncomment when enabling cookies
+// import Cookies from 'js-cookie'
+import {
+	TutorialVariant,
+	TutorialVariantOption,
+	getVariantParam,
+	getVariantPath,
+} from 'views/tutorial-view/utils/variants'
+import ButtonLink from 'components/button-link'
+import { useRouter } from 'next/router'
+import s from './variant-list.module.css'
+
+export default function VariantList({ variant }: { variant: TutorialVariant }) {
```

Comment:
We may get all of the variant data from the hook and not props, so this could change when we wire up real data.